pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -17,6 +17,7 @@
 , protobuf, speechd, libXdamage, cups
 , ffmpeg, libxslt, libxml2, at-spi2-core
 , jre
+, pipewire_0_2
```

Comment:
Haven't yet built this, but are you sure this is `pipewire_0_2` for all three channels?

I ran into some version incompatibility warnings when running pipewire 0.3 and google-chrome-stable (so I'd expect chrome to be built with `pipewire_0_2`, but `beta` and `dev` might use `pipewire`…

Reply:
We can't really check with `chromiumBeta` and `chromiumDev`, as they're broken currently on master, too: https://github.com/NixOS/nixpkgs/issues/89615

Reply:
Yes I'm fairly certain it uses pipewire_0_2. Other distros which enable this all build against pipewire 0.2 and from the chromium source it doesn't look like they've updated for 0.3.

Reply:
An easy way to test is to add the pipewire 0.3 runtime dependency to the binary packages and see if it can use it (when I try, it fails - it only works with pipewire 0.2).